packages/components/src/components/lazy/lazy.jsx

```diff
@@ -1,6 +1,6 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import UILoader from 'App/Components/Elements/ui-loader.jsx';
+import UiLoader from '../ui-loader';
```

Comment:
Please use the direct reference to the `.jsx` file to avoid duplicate style importing issue.